packages/react-storage/src/components/StorageBrowser/store/validateStoreProps.ts

```diff
@@ -0,0 +1,112 @@
+import { templateJoin } from '@aws-amplify/ui';
+
+import { LocationData } from '../actions';
+import { LocationValue, StoreProviderProps } from './types';
+
+export const CONLICTING_PROPS =
+  '`StorageBrowser` has been been provided with both `value` and `defaultValue` props. StorageBrowser must be either controlled or uncontrolled (specify either the `value` prop, or the `defaultValue` prop, but not both). Decide between using a controlled or uncontrolled `StorageBrowser` and remove one of these props.';
+
+export const READONLY =
+  'A `value` prop has been provided to `StorageBrowser` without `onValueChange`. This will render a read-only `StorageBrowser`. If the `StorageBrowser` should be mutable use `defaultValue`, otherwise set `onValueChange`.';
+
+export const MISSING_REQUIRED =
+  'A `value` prop has been provided to `StorageBrowser` without all required `location` parameters. `StorageBrowser` will ignore the provided `location`. Missing `location` parameters: %s.';
+
+export const DEPRECATED_SUFFIX =
+  '`actionType`, `location` and `path` props have been deprecated amd will be removed in a future major version. For controlled behavior provide the `value` prop or for uncontrolled behavior provide the `defaultValue` prop.';
```

Comment:
```suggestion
  '`actionType`, `location` and `path` props have been deprecated and will be removed in a future major version. Provide the `value` prop for controlled behavior or the `defaultValue` prop for uncontrolled behavior.';
```

Reply:
Thansk for catching, will fix in a follow up